lib/matplotlib/tests/test_lines.py

```diff
@@ -183,6 +184,14 @@ def test_marker_fill_styles():
     ax.set_xlim([-5, 155])
 
 
+def test_markerfacecolor_fillstyle():
+    """Test that markerfacecolor does not override fillstyle='none'."""
+    l, = plt.plot([1, 3, 2], marker=MarkerStyle('o', fillstyle='none'),
+                  markerfacecolor='red')
+    assert l.get_fillstyle() == 'none'
+    assert mcolors.same_color(l.get_markerfacecolor(), 'red')
```

Comment:
I'm confused; shouldn't the change to `_get_markerfacecolor` mean this returns `'none'`?

Reply:
Indeed. Fixed the test.

Note: `same_color()` does not work with 'none'. It's always True because `to_rgba_array()` converts 'none' to a shape (0, 4) array. Will fix that in a separate PR.